src/passes/Asyncify.cpp

```diff
@@ -1759,7 +1759,8 @@ struct Asyncify : public Pass {
       String::Split::NewLineOr(","));
     auto asserts = hasArgument("asyncify-asserts");
     auto verbose = hasArgument("asyncify-verbose");
-    auto relocatable = hasArgument("asyncify-relocatable");
+    auto importGlobals = hasArgument("asyncify-relocatable");
```

Comment:
```suggestion
    auto importGlobals = hasArgument("asyncify-relocatable");
```
Maybe `relocatable => import-globals` for symmetry?

Reply:
Done.  I left `relocatable` as an alias for now until emscripten can be updated.